build/repo.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <ItemGroup>
+    <ExcludeFromTest Include="$(RepositoryRoot)test\Microsoft.AspNetCore.Server.Kestrel.Performance\*.csproj" />
```

Comment:
Intentional. It has to match the way KoreBuild uses globs because of https://github.com/Microsoft/msbuild/issues/1576